models/demo.ipynb

```diff
@@ -0,0 +1,318 @@
+    "    AveragePooling2D,\n",
+    "    BatchNormalization,\n",
+    "    Conv2D,\n",
+    "    Dense,\n",
+    "    GlobalAveragePooling2D,\n",
+    "    ReLU,\n",
+    "    Softmax,\n",
+    ")\n",
+    "from tensorflow.keras import Model\n",
+    "from tensorflow.keras.datasets import mnist\n",
+    "from tensorflow.keras.utils import to_categorical\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "import tensorflow as tf"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# load MNIST dataset\n",
+    "(X_train, y_train), (X_test, y_test) = mnist.load_data()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# convert y_train and y_test to one-hot encoding\n",
+    "y_train = to_categorical(y_train)\n",
+    "y_test = to_categorical(y_test)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# reshape X_train and X_test to 4D tensor\n",
+    "X_train = X_train.reshape(X_train.shape[0], 28, 28, 1)\n",
+    "X_test = X_test.reshape(X_test.shape[0], 28, 28, 1)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "_input = Input(shape=(28,28,1))\n",
+    "_out = Conv2D(4, 3, use_bias=False)(_input)\n",
+    "_out = BatchNormalization()(_out)\n",
+    "_out = ReLU()(_out)\n",
+    "_out = AveragePooling2D()(_out)\n",
+    "_out = Conv2D(16, 3, use_bias=False)(_out)\n",
+    "_out = BatchNormalization()(_out)\n",
+    "_out = ReLU()(_out)\n",
+    "_out = AveragePooling2D()(_out)\n",
+    "_out = GlobalAveragePooling2D()(_out)\n",
+    "_out = Dense(10, activation=None)(_out)\n",
+    "_out = Softmax()(_out)\n",
+    "model = Model(_input, _out)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Model: \"model\"\n",
+      "_________________________________________________________________\n",
+      " Layer (type)                Output Shape              Param #   \n",
+      "=================================================================\n",
+      " input_1 (InputLayer)        [(None, 28, 28, 1)]       0         \n",
+      "                                                                 \n",
+      " conv2d (Conv2D)             (None, 26, 26, 4)         36        \n",
+      "                                                                 \n",
+      " batch_normalization (BatchN  (None, 26, 26, 4)        16        \n",
+      " ormalization)                                                   \n",
+      "                                                                 \n",
+      " re_lu (ReLU)                (None, 26, 26, 4)         0         \n",
```

Comment:
Had to avoid using lambda layer or any custom layers (layers undefined in `tfjs`) in model architecture.

Reply:
Just omitted `Lambda` layer